src/pages/ClerkScreen.tsx

```diff
@@ -188,6 +280,172 @@ const ClerkScreen = ({
           </Prose>
         )}
       </Sidebar>
+      <Modal
+        isOpen={isSystemDateModalActive}
+        centerContent
+        content={
+          <Prose textCenter>
+            <h1>
+              {weekdayAndDate(systemDate.toString())},{' '}
+              {twelveHourTime(systemDate.toString())}
+            </h1>
+            <div>
+              <p>
+                <InputGroup as="span">
+                  <Select
+                    value={systemDate.getFullYear()}
+                    data-label="year"
+                    disabled={isSavingDate}
+                    onBlur={updateSystemTime}
+                    onChange={updateSystemTime}
+                  >
+                    <option value="" disabled>
+                      Year
+                    </option>
+                    {[...Array(11).keys()].map((year) => (
+                      <option key={year} value={year + 2020}>
+                        {year + 2020}
```

Comment:
```suggestion
                      <option key={year} value={year + new Date().getFullYear()}>
                        {year + new Date().getFullYear()}
```

Reply:
If the year was set to 2030, then with this it would be impossible to change the year back to 2020, right?

Reply:
Ah, good point. Carry on.